desktop-widgets/preferences/preferences_defaults.ui

```diff
@@ -14,17 +14,24 @@
    <string>Form</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
-
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Font for lists and tables</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_1">
-      <property name="margin">
+      <property name="leftMargin">
+       <number>5</number>
+      </property>
+      <property name="topMargin">
+       <number>5</number>
+      </property>
+      <property name="rightMargin">
+       <number>5</number>
+      </property>
+      <property name="bottomMargin">
```

Comment:
this is the annoying noise that different versions of the Qt tools add... frustrating, but not your fault.